app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -78,7 +79,6 @@ class PipelineSampleReport extends React.Component {
     const cachedIncludedCategories = Cookies.get("includedCategories");
     const cachedIncludedSubcategories = Cookies.get("includedSubcategories");
 
-    const cached_name_type = Cookies.get("name_type");
```

Comment:
We had to move the "name_type" state into the parent component of PipelineSampleReport (SampleView) because the CoverageVizSidebar needed it.